README.md

```diff
@@ -97,8 +120,8 @@ app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({extended: true}))
 ```
 
-```text
-Your answer...
+```
+This is a block of code that allows Express to process "post" methods, without it forms will not be able to updated or created as post is used for both "put" and "create" methods.
 ```
 
```

Comment:
This is very close.
`app.use(bodyParser.urlencoded({extended: true}))` makes form submitted-data available under `req.body`
`app.use( bodyParser.json() )` allows fulfillment of post and put requests that carry data in json form.